tests/libtest/test_user.py

```diff
@@ -0,0 +1,20 @@
+import logging
+import os
+from time import sleep
+
+from ocs_ci.utility.utils import exec_cmd
+
+logger = logging.getLogger(__name__)
+
+
+def test_user_creation(user_factory):
+    user = user_factory()
```

Comment:
Use `username, password = user_factory()` instead.

Reply:
I will add it in next PR